build.gradle.kts

```diff
@@ -0,0 +1,114 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+  val kotlinVersion = "1.3.71"
+  `java-library`
+  `maven-publish`
+
+  kotlin("jvm") version kotlinVersion
+  kotlin("plugin.spring") version kotlinVersion
+
+  id("com.ekino.oss.plugin.kotlin-quality") version "1.0.0"
+  id("org.jetbrains.dokka") version "0.9.18"
+}
+
+java.sourceCompatibility = JavaVersion.VERSION_1_8
+
+repositories {
+  mavenCentral()
+  jcenter()
+  mavenLocal()
+}
+
+dependencies {
+  implementation(platform("org.springframework.boot:spring-boot-dependencies:${property("spring-boot.version")}"))
+
+  implementation("org.springframework.boot:spring-boot-starter-web")
+
+  implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
+  implementation("org.jetbrains.kotlin:kotlin-reflect")
+  implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+
+  implementation("io.github.microutils:kotlin-logging:${property("kotlin-logging.version")}")
+
+  implementation("com.squareup.retrofit2:retrofit:${property("retrofit.version")}")
+  implementation("com.squareup.retrofit2:converter-jackson:${property("retrofit.version")}")
+
```

Comment:
what would you say if we imported  `annotationProcessor group: "org.springframework.boot:spring-boot-configuration-processor"` ? it will help the library user with properties auto-completion

Reply:
Yes I added it good idea